README.md

```diff
@@ -62,214 +61,73 @@ Group Project Navigator is always displayed and provides quick access to most co
-
-1. Set up environment configuration variables (covered earlier)
-2. Author the XBlock in Studio
-3. Configure Group Project in Apros (3rd party LMS) 
-
-## Authoring
-
-Group Project XBlock v2 relies heavily on "nested XBlocks" feature, provided by edX Platform and XBlocks ecosystem.
-To be precise, Group Project is built from smaller XBlocks, each implementing some smaller subset of Group project 
-functionality: peer grading, submissions upload, team discussions, etc. Most of these features are internal to Group 
-Project XBlock v2 package, but some more advanced features reuse other XBlocks.
-
-Overall, the project structure is the following:
-
-* Group Project XBlock
-    * Group Activity XBlock 1
-        * Stage XBlock 1
-        * Stage XBlock 2
-            * Stage component XBlock
-            * Stage component XBlock
-        * Stage XBlock 3
-    * Group Activity XBlock 2
-        * Stage XBlock 4
-            * Stage Component XBlock
-        * Stage XBlock 5
-    * Project Navigator XBlock
-        * Project Navigation View Xblock
-        * Submissions View XBlock
-        * Resources View XBlock
-        * Private Discussion View XBlock
-    * Discussion XBlock
-    
-In this example Group Project with two activities, Project Navigator and Private Discussions is shown. Activity 1 
-is composed of three stages, Activity 2 is composed of two stages. Some stages have stage components - even smaller
-building blocks, varied by stage type. Project Navigator includes most of the child blocks that are available for it.
-
-Typical Group Project contains several activities, each of which has multiple stages. In addition to that,
-Group Project Navigator is a required component, so if it is omitted Group Project XBlock will display validation errors 
-in Studio and will display an error message in LMS and Apros. Also, Navigation View is a required component of Project
-Navigator, so any correct Group Project XBlock will contain at least Project Navigator with Navigation View in it.
-
-Activities act as larger chunks of group project by combining stages into a logically bound units. Grading 
-happens at activity level.
-
-Each stage represents single group project step - getting acquainted with the team, familiarizing with the task, 
-uploading deliverable, providing and receiving grading, etc. Stages are usually built from multiple stage components -
-from simple HTML blocks to more sophisticated Project Team and Submission blocks. Also, by using Resource blocks, stage 
-can contribute various resources to the project: grading rubrics, submission templates, or any other relevant documents
-and video.
+Students are graded by `Peer Grading` stage. This stage contains questions, some of these questions can be descriptive, 
+other should contain numerical evaluation of group submissions. Descriptive feedback can be displayed to students in
+`Grade Display` stage, while numerical feedback can be used to calculate grades.
```

Comment:
I'm not sure I understand what this means. I had a look in the detailed documentation but I didn't find an explanation there. What is a descriptive question? What are descriptive feedback and numerical feedback?